pkg/term/term_windows.go

```diff
@@ -48,8 +48,8 @@ func GetWinsize(fd uintptr) (*Winsize, error) {
 
 	// TODO(azlinux): Set the pixel width / height of the console (currently unused by any caller)
 	return &Winsize{
-		Width:  uint16(info.Window.Bottom - info.Window.Top + 1),
-		Height: uint16(info.Window.Right - info.Window.Left + 1),
+		Width:  uint16(info.Window.Right - info.Window.Left + 1),
+		Height: uint16(info.Window.Bottom - info.Window.Top + 1),
```

Comment:
That wrapping should be solved by https://github.com/Microsoft/docker/commit/7c83ec1ec94814f771b2d3c44519e93d5de51f29 which @brendandixon or @ahmetalpbalkan where going to open a PR for.